src/mscorlib/shared/System/Lazy.cs

```diff
@@ -183,6 +183,7 @@ internal static LazyThreadSafetyMode GetModeFromIsThreadSafe(bool isThreadSafe)
     /// </para>
     /// </remarks>
     [Serializable]
+    [ComVisible(false)]
```

Comment:
Undo this. It is a recent change made by somebody else - this should be deleted in the CoreRT copy instead.

(It is an example of change that the mirror would take care of propagating if we had one.)